cmd/protoc-gen-loggingtags/README.md

```diff
@@ -0,0 +1,54 @@
+# protoc-gen-loggingtags
+
+A `protoc` plugin that generates `LoggingTags() map[string]interface{}`
+functions on Go proto message structs based on field options inside the source
+`.proto` files. The logging tags returned are safe to log without leaking
+information. The logging tag functions are code-generated so it's always in
+sync with the source files.
+
+The `grpcserver` package automatically checks if request or response messages
+implement the LoggingTags method and includes the returned tags in the context
+tags for logging.
+
+## Example
+
+```go
+syntax = "proto3";
+import "github.com/heroku/x/cmd/protoc-gen-loggingtags/heroku/loggingtags/safe.proto";
```

Comment:
Runtime imports the proto as `heroku/loggingtags/safe.proto`, and then rewrites the import via flags on protoc. It's more common ([example](https://github.com/mwitkow/go-proto-validators#installing-and-using)) to specify the full package import and set the protoc path to GOPATH or vendor, but that produces a very long (and repetitive) import path here.

Perhaps we should reorganize now, so the import is something like `github.com/heroku/x/cmd/protoc-gen-loggingtags/safe.proto`?

Reply:
Would it help anything to have the proto somewhere else in the tree other than under cmd/protoc-gen-loggingtags? Either way, probably ending up with whatever requires the least config when consuming would be preferable.